src/components/game/index.jsx

```diff
@@ -1,9 +1,67 @@
-import React from "react";
+import React, { useState } from "react";
 import Board from "components/game/Board";
 
+// Calculates who won the game
+function calculateWinner(board) {
+  // All the winning combinations
+  const winningConditions = [
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+    [0, 3, 6],
+    [1, 4, 7],
+    [2, 5, 8],
+    [0, 4, 8],
+    [2, 4, 6]
+  ];
+
+  for (let i = 0; i < winningConditions.length; i++) {
+    const [a, b, c] = winningConditions[i];
+    // If the values of each index match, return the winner
+    if (board[a] && board[a] === board[b] && board[a] === board[c]) { 
+      return board[a];
+    }
+    // Else if there are no more null values in the board, return "Draw"
+    else if (!board.includes(null)) {
+      return "Draw";
+    }
+  }
+  return null;
+}
+
 // Renders the Board component
 function Game() {
-  return <Board />;
+  const [valuesArr, setValuesArr] = useState(Array(9).fill(null)); // Values of the 1st-9th square
+  const [nextPlayer, setNextPlayer] = useState("O"); // Initial player aka first player to go is O
+  const winner = calculateWinner(valuesArr);
```

Comment:
`calculateWinner` gets called every `rerender` which is fine for this project. But for bigger ones with many components and a huge board, this might introduce performance issues.

A better way to handle this is to put `calculateWinner` in `updateValuesArr` and make `winner` a state so you can pass it to the `Board` as a prop.